### PR TITLE
Fix jballerina-unit-test test failures in windows build

### LINKED ISSUE #19314: Fix windows tests failures
**Description:**
These gradle projects have test failures in Travis windows build.

- [x] language-server:language-server-core 
- [ ] ballerina-packerina 
- [ ] ballerina-lang 
- [x] ballerina-http 
- [x] ballerina-file 
- [x] ballerina-task 
- [x] ballerina-socket 
- [ ] jballerina-unit-test 
- [ ] jballerina-integration-test 
- [ ] plugin-vscode

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by NipunaMadhushan**
## Purpose
Enable some tests in github actions windows build

Fixes #19314

## Approach
Fix ClassPathNotFoundException by replacing the classloader with jarLibrary.path()
Fix AssertionError by replacing "\n" with System.lineSeperator() 
Fix InvalidPathException by removing invalid characters in the filepath

### CODE REVIEW

File: tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java [hunk 1]
```diff
@@ -1329,13 +1332,19 @@ public static ExitDetails run(CompileResult compileResult, String... args) {
                 packageManifest.name().toString(),
                 packageManifest.version().toString(),
                 MODULE_INIT_CLASS_NAME);
-        URLClassLoader classLoader = (URLClassLoader) compileResult.getClassLoader();
+        //URLClassLoader classLoader = (URLClassLoader) compileResult.getClassLoader();
```
Comment (azinneera) [inline tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java:1335]: We can remove this commented line

File: tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java [hunk 2]
```diff
@@ -1329,13 +1332,19 @@ public static ExitDetails run(CompileResult compileResult, String... args) {
...
+        Collection<JarLibrary> jarPathRequiredForExecution = compileResult.getJarPathRequiredForExecution();
+        String classpath1 = "";
```
Comment (azinneera) [inline tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java:1337]: Let's rename this variable properly. i.e: `classpath`

File: tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java (hunk 1 above)
Comment (azinneera) [reply tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java:1335]: Please clean up other places too

File: tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java [hunk 3]
```diff
@@ -1329,13 +1332,19 @@ public static ExitDetails run(CompileResult compileResult, String... args) {
...
+        for (JarLibrary jarLibrary:jarPathRequiredForExecution) {
+            classpath1 = classpath1 + File.pathSeparator + jarLibrary.path();
+        }
 
         try {
             final List<String> actualArgs = new ArrayList<>();
             actualArgs.add(0, "java");
             actualArgs.add(1, "-cp");
-            String classPath = System.getProperty("java.class.path") + ":" + getClassPath(classLoader);
+            //String classPath = System.getProperty("java.class.path") + File.pathSeparator + getClassPath(classLoader);
```
Comment (azinneera) [inline tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java:1346]: We can remove this line

File: tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java [hunk 4]
```diff
@@ -1329,13 +1332,19 @@ public static ExitDetails run(CompileResult compileResult, String... args) {
                 packageManifest.name().toString(),
                 packageManifest.version().toString(),
                 MODULE_INIT_CLASS_NAME);
-        URLClassLoader classLoader = (URLClassLoader) compileResult.getClassLoader();
+        //URLClassLoader classLoader = (URLClassLoader) compileResult.getClassLoader();
+        Collection<JarLibrary> jarPathRequiredForExecution = compileResult.getJarPathRequiredForExecution();
+        String classpath1 = "";
+        for (JarLibrary jarLibrary:jarPathRequiredForExecution) {
```
Comment (hemikak) [inline tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java:1338]: Need formatting

Comment (NipunaMadhushan) [reply tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java]: Corrected it

**Comment by riyafa** [comment]
LGTM

**Review by keizer619 — DISMISSED** [review]

**Pull request closed without merging**